src/components/Page/components/Header/Header.scss

```diff
@@ -52,57 +50,41 @@ $action-menu-rollup-computed-width: rem(40px);
 }
 
 .BreadcrumbWrapper {
-  flex: 0 1 auto;
+  flex: 0 0 auto;
+  grid-area: breadcrumbs;
   max-width: 100%;
-
-  &.newDesignLanguage {
-    grid-area: breadcrumbs;
-    flex: 0 0 auto;
-    margin-right: spacing();
-  }
+  margin-right: spacing();
 
   @include print-hidden;
 }
 
 .PaginationWrapper {
-  flex: 0 0 auto;
-  margin-left: auto;
+  display: flex;
+  justify-content: flex-end;
+  margin-left: spacing(extra-tight);
   line-height: 1;
 
-  .newDesignLanguage & {
-    display: flex;
-    justify-content: flex-end;
-    margin-left: spacing(extra-tight);
-    // stylelint-disable-next-line selector-max-combinators
-    button {
-      border: 1px solid var(--p-border-neutral-subdued);
-      box-shadow: none;
-      // stylelint-disable-next-line selector-max-combinators, selector-max-specificity
-      &:hover,
-      &:active,
-      &:focus {
-        border: 1px solid var(--p-border-neutral-subdued);
-      }
+  button {
+    // stylelint-disable declaration-no-important
+    border: 1px solid var(--p-border-neutral-subdued) !important;
+    box-shadow: none !important;
```

Comment:
I assume the `!important` is needed now, is there a way we can clean this up. Happy to ship and fix later.

Reply:
It's needed because there was specificity before with the added class. Not sure we can remove it and not sure that re-working button will fix it either. I'm fairly fine with leaving !important for now though since this is wrapped in `.PaginationWrapper`. Should be fairly safe